src/mumble/AudioOutputSample.cpp

```diff
@@ -124,9 +124,12 @@ AudioOutputSample::AudioOutputSample(const QString &name, SoundFile *psndfile, b
 	sfHandle = psndfile;
 	iOutSampleRate = freq;
 
-	// Check if the file is good
-	if (sfHandle->channels() <= 0 || sfHandle->channels() > 2) {
-		sfHandle = NULL;
+	if (sfHandle->channels() == 1) {
+		bStereo = false;
+	} else if (sfHandle->channels() == 2) {
+		bStereo = true;
+	} else {
+		sfHandle = NULL;  // sound file is corrupted
```

Comment:
Since we're requiring cpp11 for Mumble 1.4.0, I think you should be using `nullptr` instead of `NULL`

Reply:
Hummmm... I will replace all NULL in these files.

Reply:
Or do you think it is better to do this kind of replacement for all source files in a separate commit?

Reply:
A separate commit is probably cleaner. But if you have used `NULL` somewhere else, feel free to replace that as well :)